internal/core/items/text.rs

```diff
@@ -123,6 +125,15 @@ impl Item for ComplexText {
         (*backend).draw_text(self, self_rc, size, &self.cached_rendering_data);
         RenderingResult::ContinueRenderingChildren
     }
+
+    fn bounding_rect_for_geometry(
+        self: core::pin::Pin<&Self>,
+        _window_adapter: &Rc<dyn WindowAdapter>,
+        _self_rc: &ItemRc,
+        geometry: LogicalRect,
+    ) -> LogicalRect {
+        geometry
```

Comment:
Should that not be the same as for the `SimpleText` item?

Reply:
Oops, yes, good catch!